mne/channels/channels.py

```diff
@@ -613,7 +618,10 @@ def add_channels(self, add_list, copy=None):
 
         # Create final data / info objects
         data = np.concatenate(data, axis=con_axis)
-        infos = [self.info] + [inst.info for inst in add_list]
+        infos_to_add = deepcopy([inst.info for inst in add_list])
+        if force_update_info is True:
+            _force_update_info(self.info, infos_to_add)
+        infos = [self.info] + infos_to_add
         new_info = _merge_info(infos)
```

Comment:
to me it's better to keep the update logic in one place. You can keep
your _force_update_info
function but call it from _merge_info